public/views/modals/address-definition.html

```diff
@@ -6,14 +6,15 @@
   </section>
   <section class="middle tab-bar-section">
     <h1 class="title ellipsis" style="color: #4E4E4E !important" translate>
-      Definition of smart address
+      Definition of smart contract
     </h1>
   </section>
 </nav>
 
 
 <div class="modal-content fix-modals-touch">
-	<div class="row columns m20t enable_text_select" style="word-break: break-all;">{{address}} <span translate>can be spent if:</span></div>
+	<div class="row columns m20t center enable_text_select" style="word-break: break-all;">{{address}}</div>
+	<div class="row columns m20t center enable_text_select" style="word-break: break-all;"><span translate>can be spent if:</span></div>
```

Comment:
Introducing manual line-breaks is not a good idea given that the page should be displayed on devices with various widths. `style="overflow-wrap: break-word"` instead of `style="word-break: break-all;"` should solve the problem of weird word-breaking.